kong/plugins/prometheus/exporter.lua

```diff
@@ -23,24 +23,24 @@ local function init()
 
   prometheus = require("kong.plugins.prometheus.prometheus").init(shm, "kong_")
 
-  -- accross all services
+  -- across all services
   metrics.connections = prometheus:gauge("nginx_http_current_connections",
-                                         "Number of HTTP connections",
```

Comment:
The formatting used here is borrowed from style guide we use in https://github.com/kong/kong.
Can we revert this back and introduce the following in .luacheckrc?
```
max_line_length = false
```

Reply:
eek, I didn't know that was in our style guide :(
I suppose it would be too much effort to change the style guide at this point?

Reply:
> I suppose it would be too much effort to change the style guide at this point?

Yeah, I doubt we will be changing that style guide at this point.